package.json

```diff
@@ -25,7 +26,7 @@
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.57",
     "@popperjs/core": "^2.10.2",
-    "@rjsf/core": "^3.0.0",
+    "@rjsf/core": "^4.0.1",
```

Comment:
Breaking changes for RJSF v4 relate to the material-ui styled form, which we don't use